src/configs.py

```diff
@@ -143,11 +161,12 @@ def verify_schedule_credentials(cls, values):
         if values["schedule_file"] is None:
             return values
         # A valid schedule file is given; schedule-credentials are thus required:
-        c_secret, c_id, t_id = values["client_secret"], values["client_id"], values["tenant_id"]
-        if None in [c_secret, c_id, t_id]:
+        c_secret, c_id, token_url = values["client_secret"], values["client_id"], values["token_url"]
+        if None in [c_secret, c_id, token_url]:
```

Comment:
Can `token_url` ever be `None`?

Reply:
Actually not, good catch. Will remove the check